GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -70,10 +78,19 @@ function CopyBinariesToInstall()
     copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${PRJFSDESTINATION}/.\""
     eval $copyPrjFS || exit 1
     
+    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${LIBRARYEXTENSIONSDESTINATION}/.\""
+    eval $copyPrjFS || exit 1
+    
     currentDirectory=`pwd`
+    
     cd "${STAGINGDIR}/usr/local/bin"
     linkCommand="ln -sf ../vfsforgit/gvfs gvfs"
     eval $linkCommand
+    
+    cd "${STAGINGDIR}/$LIBRARYDESTINATION"
+    linkCommand="ln -sf ../vfsforgit/prjfs/libPrjFSLib.dylib libPrjFSLib.dylib"
```

Comment:
Why not just install `libPrjFSLib.dylib` the the `usr/local/vfsforgit` directory (at least for now)?

At this point ProjFS is not ready for consumption by any other applications.  Until we have a design in place for shipping ProjFS on Mac as its own project, I don't think we should drop it (or links to it) in `usr/local/lib`.

(This is also consistent with Windows, VFS4G does not install the ProjFS native DLL to the system32 directory, it's side-by-side with GVFS.Mount.exe)

Reply:
If the reason is the library search path in the executable, I believe we can change that.

Reply:
I agree with William. I think the default library search path should include `.` so we should be able to pick up the lib next to the executable. But if it's not, I think we should change that as @pmj suggests rather than making a link here.

Reply:
updated to install `libPrjFSLib.dylib` in `/usr/local/vfsforgit`